connection.go

```diff
@@ -2,210 +2,22 @@
 package ucloud
 
 import (
-	"crypto/sha1"
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
 	"net/http"
-	URL "net/url"
-	"reflect"
-	"sort"
-	"strconv"
-	"strings"
+	"time"
 )
 
-type UResponse interface {
-	OK() bool
-	Msg() string
-	Data() interface{}
-	SetCode(int)
-}
-
-type URequest interface {
-	// This should return new instance for json Unmarshal
-	R() UResponse
-}
-
-type BaseResponse struct {
-	RetCode int
-	Action  string `json:",omitempty"`
-	Message string `json:",omitempty"`
-}
-
-func (b *BaseResponse) OK() bool {
-	return (b.RetCode == 0) // XXX ReturnCode is string for abnormal
-}
-func (b *BaseResponse) SetCode(i int) {
-	b.RetCode = i // This is a patch for Ucloud API return with string
-}
-
-func (b *BaseResponse) Msg() string {
-	return b.Message
-}
-
 type UcloudApiClient struct {
-	baseURL    string
+	baseUrl    string
```

Comment:
thanks for remind :)

On Fri, May 6, 2016 at 3:41 PM, Meng Zhuo <notifications@github.com> wrote:

> In connection.go
> <https://github.com/mengzhuo/ucloud-go-sdk/pull/3#discussion_r62297703>:
>
> >  type UcloudApiClient struct {
> > -	baseURL    string
> > +	baseUrl    string
>
> URL is short for Unique Resource Location not Url
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mengzhuo/ucloud-go-sdk/pull/3/files/11ac4a16928d228ae9604e3a1dfbfcdbaa1c49e2#r62297703>
>



-- 
夏海明